backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -23,28 +24,32 @@ export const updateClusterSettings = async (
 ): Promise<{ success: boolean; error: string }> => {
   const coreV1Api = fastify.kube.coreV1Api;
   const namespace = fastify.kube.namespace;
-  const query = request.query as { [key: string]: string };
+  const body = request.body as { [key: string]: string | NotebookTolerationSettings };
```

Comment:
There are too many `as` in this function, we should avoid doing that. Let's do a simple refactor.
1. Change the request type from `FastifyRequest` to `FastifyRequest<{ Body: ClusterSettings }>`
2. Change `const body = request.body as { [key: string]: string | NotebookTolerationSettings };` to `const { pvcSize, cullerTimeout, userTrackingEnabled, notebookTolerationSettings} = request.body` because they are used a lot of times below
3. Avoid using something like `userTrackingEnabled as string` here, instead, we can just use `String(userTrackingEnabled)` every time when you see a type warning